python/ray/runtime_env.py

```diff
@@ -261,7 +261,7 @@ class MyClass:
         py_modules (List[URI]): List of URIs (either in the GCS or external
```

Comment:
This class seems to have wrong information. (e.g., pip type is wrong from__init__ & docstring still contains the type.)